### DISCUSSION TITLE
WireGuard mode in mitmproxy where find client config file? How work with it? Any documentation?

### DISCUSSION THREAD

**Discussion by Vasa211** [mitmproxy/mitmproxy#6824] [General]
#### Problem Description
WireGuard mode in mitmproxy where find client config file? (not mitmweb in "mitmproxy --mode wireguard@1111")

#### Steps to reproduce the behavior:
1. Look in ~./wireguard but nothing

#### System Information
Using last mitmproxy

And how it work? i cant find any documentation I need start wg too (with mitm), or setup some networks? what i need do for start mitmproxy in WireGuard mode on my server, share config to my iphone and see all requests (get, post, put, patch)

**Comment by mhils** [comment]
You can find the WireGuard config in the mitmproxy/mitmdump event log (or straight in mitmweb). We still need to do some UI work here.

**Reply by shiwenwen** [reply]
I have the same question, how do I configure my own conf, how do I configure multi-clients
